src/components/timeselector/TimeSelectorDirective.js

```diff
@@ -247,11 +247,19 @@ goog.require('ga_time_service');
               var year = scope.options.years[i];
               year.available = false;
               olLayers.forEach(function(olLayer, opt) {
-                if (year.available || !olLayer.bodId) {
+                if (year.available ||
+                    (!olLayer.bodId &&
+                     !gaMapUtils.isExternalWmtsLayer(olLayer.id))) {
```

Comment:
May be the test of `olLayer.timeEnabled` is enough instead of `(!olLayer.bodId && !gaMapUtils.isExternalWmtsLayer(olLayer.id))`